build/godep-builder/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM golang
+FROM golang:latest
```

Comment:
```suggestion
FROM golang:1.12
```

Reply:
I don't think this is a tiny nit at all. I really like this suggestion--thanks a lot for reviewing. I made the change